website/src/libs/api-v3/collectionManipulators.js

```diff
@@ -0,0 +1,20 @@
+import { findIndex } from 'lodash';
+
+export function removeElementFromArray (array, element) {
```

Comment:
I think you're right. I've been struggling to find a good name for this. I think `removeFromArray` does the job.